packages/sveltekit/src/vite/sourceMaps.ts

```diff
@@ -109,6 +117,20 @@ export async function makeCustomSentryVitePlugin(options?: SentryVitePluginOptio
       }
     },
 
+    transform: async (code, id) => {
```

Comment:
Thinking out loud: We should probably move to a virtual file solution here in the long term - reason being that we inject at the end of files right now which may not be in time if errors are thrown above the snippet. Since esm imports are hoisted to the top of a file via spec we can basically assume that the snippet runs before any user code.